controllers/replicationdestination_controller.go

```diff
@@ -46,6 +46,9 @@ const (
 // RsyncContainerImage is the container image name of the rsync data mover
 var RsyncContainerImage string
 
+// RcloneContainerImage is the container image name of the rclone data mover
+var RcloneContainerImage string = "quay.io/backube/scribe-mover-rclone:latest"
```

Comment:
will this image name eventually come from a config file?

Reply:
Related to #64 . It'll eventually be settable via env var to the operator plus hopefully default to the proper release tag.